packages/did-ethr-resolver/src/implementations/operator.ts

```diff
@@ -77,6 +81,14 @@ export class Operator extends Resolver implements IOperator {
     return this._keys.publicKey;
   }
 
+  private setOwner(privateKey: string, providerUrl?: string): IdentityOwner {
+    if (providerUrl) {
+      const provider = getDefaultProvider(providerUrl);
+      return new Wallet(privateKey, provider);
+    }
+    return new Wallet(privateKey);
+  }
```

Comment:
If we can just define an `IdentityOwner` using `new Wallet`, do we even need the `withKey` and `withSigner` functions that are in `did-ethr-resolver`?

Reply:
@JGiter if you have any insight on the above ☝️ , that would be great